man/make.cards.Rd

```diff
@@ -1,54 +1,75 @@
+  Full item wordings are printed on the front of business cards, item identifiers (ID) on the back.
 }
 \usage{
-  make.cards(q.set, study.language, paper.format, output.pdf, manual.lookup)
+  make.cards(
+  	q.set,
+  	study.language=NULL,
+  	paper.format="AveryZweckformC32010.Rnw",
+  	output.pdf=TRUE,
+  	manual.lookup=NULL,
+    wording.font.size = NULL,
+    file.name = "QCards"
+    )
 }
 
 \arguments{
   \item{q.set}{
-    A matrix with short handles as rownames, languages in columns, as produced by \code{\link{build.q.set}}.
+    A matrix with short handles as rownames, languages (if applicable) in columns, as produced by \code{\link{build.q.set}}.
   }
   \item{study.language}{
   	A character vector of length one, choosing a language from \code{q.concourse.dir} and same as in \code{\link{import.q.concourse}} to print items in.
+  	Defaults to \code{NULL}, in which case the first column from \code{q.concourse} is selected.
   }
   \item{paper.format}{
   	A character vector of length one, choosing among available templates of business card sheets.
-  	Currently only \code{"AveryZweckformC32010"} is available.
+  	Defaults to the only currently available \code{"AveryZweckformC32010.Rnw"}.
+    Must include file extension of template.
   }
   \item{output.pdf}{
   	Logical.
   	If \code{TRUE}, function invokes \code{knit2pdf} to create a PDF in the workspace.
-  	If \code{FALSE}, function invokes \code{knit} to return a TEX in the workspace, may be preferable if no TEX installation is available on the used instance.
-  	Defaults to true.
+  	If \code{FALSE}, function invokes \code{knit} to return only a .TEX in the workspace, may be preferable if no \href{http://www.latex-project.org}{LaTeX} installation is available on the used computer.
+  	Defaults to \code{TRUE}.
   }
   \item{manual.lookup}{
   	A matrix with meaningful short handles (same as in \code{\link{build.q.set}}, \code{\link{import.q.concourse}}) as rownames, and arbitary, unique identifying strings in any of the columns as also expected in \code{\link{import.q.sorts}} and \code{\link{import.q.feedback}}.
-		Defaults to NULL in which case items are automatically identified by full item hashes, as also detected by \code{\link{import.q.sorts}} and \code{\link{import.q.feedback}}.
+		Defaults to \code{NULL} in which case items are automatically identified by full item hashes, as also detected by \code{\link{import.q.sorts}} and \code{\link{import.q.feedback}}.
+  }
+  \item{wording.font.size}{
+    A character vector of length 1 to set the font size of the full item wording on the cards.
+    Defaults to \code{NULL} in which the default font size 12pt is used.
+    Only \href{http://en.wikibooks.org/wiki/LaTeX/Fonts#Sizing_text}{standard LaTeX font sizes} are allowed, from \code{\\tiny} to \code{\\Huge}.
+  }
+  \item{file.name}{
+    A character vector of length 1 to set the filename \emph{without file extension}.
+    Defaults to \code{QCards}.
   }
```

Comment:
on `babel.language`, link to `babel` package, or where the languages or the package is described